libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -975,6 +975,8 @@ namespace SohImGui {
                         Tooltip("The default response to Kaepora Gaebora is\nalways that you understood what he said");
                         EnhancementCheckbox("Fast Ocarina Playback", "gFastOcarinaPlayback");
                         Tooltip("Skip the part where the Ocarina playback is called when you play\na song");
+                        EnhancementCheckbox("Unlimited Ocarina Speed", "gDpadNoDropOcarinaInput");
```

Comment:
Not the biggest fan of this name. Maybe "Prevent dropped ocarina inputs" or something? Unlimited speed just doesn't really explain what it's doing.

Reply:
I think that name will work. As mentioned earlier, I wasn't sure what to call the option to begin with.

Reply:
For sure, I'm not 100% on the name idea I had either but if it works for you it works for me

Reply:
tl;dr names are hard